src/main/java/webserver/RequestHandler.java

```diff
@@ -23,9 +30,20 @@ public void run() {
                 connection.getPort());
 
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
-            // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
+
+            BufferedReader buf = new BufferedReader(new InputStreamReader(in, StandardCharsets.UTF_8));
+            String line = buf.readLine();
+            String[] token = HttpRequestUtils.separateUrl(line);
```

Comment:
```suggestion
            String url = HttpRequestUtils.getURL(line);
```
당장 필요한 부분만 따서 이렇게 하는게 더 깔끔하지 않을까 싶습니다.

Reply:
감사합니다! 
3단계 진행중 이여서 3단계에서 반영하도록 하겠습니다.